src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,93 @@
+import React, { Children, PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import bem from 'bem';
+import styles from './Tabs.scss';
+
+class Tabs extends PureComponent {
+    constructor(props) {
+        super(props);
+
+        const { activeTabId } = props;
+
+        this.state = {
+            activeTabId,
+            derivedProp: activeTabId // eslint-disable-line react/no-unused-state
+        };
+    }
+
+    static getDerivedStateFromProps(props, state) {
+        if (props.activeTabId === state.derivedProp) {
+            return null;
+        }
+        return {
+            activeTabId: props.activeTabId,
+            derivedProp: props.activeTabId
+        };
+    }
+
+    handleTabChange(e, id) {
+        e.preventDefault();
+        const { onChange } = this.props;
+        this.setState({
+            activeTabId: id
+        });
+        if (!onChange) {
+            return true;
+        }
+        return onChange(id);
+    }
+
+    render() {
+        const { activeTabId: idFromProps, children, onChange, ...rest } = this.props;
+        const { activeTabId } = this.state;
+
+        return (
+            <div {...rest} {...this.block()}>
+                <div {...this.elem('container')}>
+                    {Children.map(children, tab => {
+                        const { href, id } = tab.props;
+                        if (!id) {
+                            return null;
+                        }
+                        const active = activeTabId === id;
+                        return (
+                            <a
```

Comment:
Also, I think we should render `<Tab>` here, not `<a>`, so we should change `Tab` component correspondently.

Reply:
My reasoning to with `<a />` is because your tabs may be routed (e.g. each tab corresponds to a particular route, using React router for example). What element would you suggest it should render instead (even if it would be wrapped by `<Tab />`)?